### PR TITLE
Differentiate between reindexing and re-feed in schema reference doc

### PR DESCRIPTION
**Pull request by jonmv**
@bratseth or @geirst please review and merge

### CODE REVIEW

File: en/reference/schema-reference.html [hunk 1]
```diff
@@ -3114,12 +3114,14 @@ <h2 id="example">Example</h2>
 
 <h2 id="modifying-schemas">Modifying schemas</h2>
 <p>
-This section describes how a schema in a live application can be modified - categories:
+This section describes how a schema in a live application can be modified—categories:
```
Comment (bratseth) [inline en/reference/schema-reference.html:3117]: You utter snob!

File: en/reference/schema-reference.html [hunk 2]
```diff
@@ -3374,11 +3366,68 @@ <h3 id="changes-that-require-re-feed">Changes that require re-feed</h3>
...
         <br>
-        NOTE: If not re-feeding after such a change, serving works,
+        NOTE: If not reindexing after such a change, serving works,
         but recall is undefined as the index has been produced using one match mode
         while run-time is using a different match mode
       </td></tr>
 
+  </tbody>
+</table>
+Example: Given a content cluster <em>mycluster</em> with mode <em>index</em>:
```
Comment (bratseth) [inline en/reference/schema-reference.html:3376]: This is a reference, ideally we should write a guide doc whenever we feel a need to add lots of explanation like this.

**Review by bratseth — APPROVED** [review]
👍 👍

**Review by geirst — COMMENTED** [review]
👍

**Pull request merged**